### ISSUE TITLE
Allow teaser to accept title and standfirst markup

### ISSUE THREAD

**Issue by i-like-robots** [Financial-Times/x-dash#124]
The original `n-teaser` component allowed implementers to render unescaped values for the `title` and `standfirst` properties. This functionality has recently been removed due to concerns with rendering content sourced from 3rd parties (https://github.com/Financial-Times/n-teaser/pull/195).

However, there are cases where this feature is desirable - for example highlighting matching terms on the search page - and so it may be useful to allow an option to toggle it on.

**Comment by quarterto** [comment]
could we just make people who have this use case inject their own `Title` component? might need first class support for that as with `Link`

**Comment by i-like-robots** [comment]
Here is a temporary workaround from @chee:

```js
const { h } = require('@financial-times/x-engine');

const wrap = (html) => { dangerouslySetInnerHTML: { __html: html }  };

const teaser = { … };

const htmlTeaser = {
 ...teaser,
 standfirst: h('span', wrap(teaser.standfirst))
};
```

**Comment by i-like-robots** [comment]
Fixed in #139
